relay-dynamic-config/src/defaults.rs

```diff
@@ -30,6 +30,13 @@ const MONGODB_QUERIES: &[&str] = &["*\"$*", "{*", "*({*", "*[{*"];
 /// A list of patterns for resource span ops we'd like to ingest.
 const RESOURCE_SPAN_OPS: &[&str] = &["resource.script", "resource.css", "resource.img"];
 
+const CACHE_SPAN_OPS: &[&str] = &[
+    "cache.get_item",
+    "cache.save",
+    "cache.clear",
+    "cache.delete_item",
+];
```

Comment:
Does `cache.*` cover all operations without false positives?

Reply:
Yes it should!

Reply:
Should we simplify the patterns then?

Reply:
I don't mind being more explicit here since there is only 4 operations, it feels a bit more robust versus a wildcard on spans ops starting with cache.